ClientUpdater/Updater.cs

```diff
@@ -597,20 +604,21 @@ private static void DoVersionCheck()
                 {
                     Logger.Log("Updater: Checking version on the server.");
 
-                    WebClient client = new WebClient
+                    using WebClient client = new WebClient
                     {
-                        CachePolicy = new RequestCachePolicy(RequestCacheLevel.NoCacheNoStore),
-                        Encoding = Encoding.GetEncoding("Windows-1252")
+                        CachePolicy = new RequestCachePolicy(RequestCacheLevel.NoCacheNoStore)
                     };
 
                     client.Headers.Add(HttpRequestHeader.UserAgent, GetUserAgentString());
```

Comment:
Doesn't the `client.Headers.Add()` call disposed `WebClient` object because it isn't encapsulated in `using` statement?

Reply:
Those are actually C#8 using declarations, not classic using statements: [using statement (C# Reference)](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement).
The `{` and `}` are actually coming from the `WebClient` object initializer.

Reply:
I see. That makes more sense then.